maplibre/src/style/style.rs

```diff
@@ -33,6 +35,8 @@ impl Default for Style {
             center: Some([46.5197, 6.6323]),
             pitch: Some(0.0),
             zoom: Some(13.0),
+            maxzoom: Some(20),
+            minzoom: Some( 0), // karlos: ok??? Detault values in extra file?
```

Comment:
There should be an option in your IDE to run Rust fmt when saving the file.

Reply:
I do ussually, but your rustfmt.toml causes:
Warning: can't set `imports_granularity = Crate`, unstable features are only available in nightly channel.
Warning: can't set `group_imports = StdExternalCrate`, unstable features are only available in nightly channel.
And with commenting out the toml, noting happens.

Reply:
Oh yes, you will need nightly fmt. So you will continue to use stable Rust, but you can use nightly formatter.

Try to run: just format or just fmt

Reply:
Not found: just. Is a windows package manager? I use macOS.
rustup install nightly-x86_64-apple-darwin
But still: rustfmt 1.4.38-stable (e092d0b6 2022-07-16)
% rustup component add rustfmt --toolchain nightly
info: component 'rustfmt' for target 'x86_64-apple-darwin' is up to date  :(  no nightly-x86

Reply:
In that case try to run cargo +nightly fmt

You probably have it already installed :)

Reply:
Yes! So I have to add "+nightly". I will try to change this in the VS-Code